Adding `artifacts` section to docs (#4481)
iterative/dvc.org: content/docs/use-cases/model-registry.md

</admon>

> To make a Git-native registry (on top of DVC or not), one option is to use [GTO]

Reviewer: ```suggestion
To make a Git-native registry (with or without DVC), one option is to use [GTO]
```

(Git Tag Ops). It tags ML model releases and promotions. This creates
abstractions for your models, which lets you **manage their lifecycle** freely
and directly from Git.

And to **productionize** the models, you can save and package them with the
[MLEM] Python API or CLI, which automagically captures all the context needed to
distribute them. It can store model files on the cloud (by itself or with DVC),
list and transfer them within locations, wrap them as a local REST server, or
even containerize and deploy them to cloud providers!

> To let your teams **collaborate** on project more efficiently, you can use

Reviewer: ```suggestion
To allow your teams **collaborate** on projects more efficiently, use
```

> [Studio](/doc/studio/user-guide/model-registry/what-is-a-model-registry) web

Reviewer 2: ```suggestion
[Studio model registry](/doc/studio/user-guide/model-registry/what-is-a-model-registry)
```

> application to organize, discover, version, promote models to production and

Reviewer 2: ```suggestion
 to organize, discover, version, promote models to production and
```
